capplet/gsp-keyfile.h

```diff
@@ -46,13 +48,22 @@ gboolean gsp_key_file_get_boolean      (GKeyFile       *keyfile,
                                         gboolean        default_value);
 #define gsp_key_file_get_string(key_file, key) \
          g_key_file_get_string (key_file, G_KEY_FILE_DESKTOP_GROUP, key, NULL)
+#define gsp_key_file_get_delay(key_file) \
+         (g_key_file_get_integer (key_file, G_KEY_FILE_DESKTOP_GROUP, GSP_KEY_FILE_DESKTOP_KEY_GNOME_DELAY, NULL) + \
+         g_key_file_get_integer (key_file, G_KEY_FILE_DESKTOP_GROUP, GSP_KEY_FILE_DESKTOP_KEY_DELAY, NULL))
 #define gsp_key_file_get_locale_string(key_file, key) \
          g_key_file_get_locale_string(key_file, G_KEY_FILE_DESKTOP_GROUP, key, NULL, NULL)
 
 #define gsp_key_file_set_boolean(key_file, key, value) \
          g_key_file_set_boolean (key_file, G_KEY_FILE_DESKTOP_GROUP, key, value)
 #define gsp_key_file_set_string(key_file, key, value) \
          g_key_file_set_string (key_file, G_KEY_FILE_DESKTOP_GROUP, key, value)
+#define gsp_key_file_set_delay(key_file, value) \
```

Comment:
Do we actually handle `X-GNOME-Autostart-Delay` (or any of their other keys) anywhere else? I can't find the code...
If not, then there's no need to remove that key.